server/db/index.js

```diff
@@ -0,0 +1,87 @@
+const db = require('./database');
+const {
+  Products,
+  Categories,
+  Sizes,
+  ShoeSizes,
+  Genders,
+  Colors,
+  ProductListings
+} = require('./product_models/index');
+
+const { Orders, OrderStatuses, OrderItems } = require('./order_models/index');
```

Comment:
A little heavy handed, dont need to make everything a reference. perhaps enum?

Reply:
JL